company-lsp.el

```diff
@@ -459,11 +463,11 @@ See the documentation of `company-backends' for COMMAND and ARG."
      ;; actions only apply to the pre-completion content. We backup the current
      ;; prefix and restore it after company completion is done, so the content
      ;; is restored and textEdit actions can be applied.
-     (or (company-lsp--cache-item-candidates (company-lsp--cache-get arg))
+     (or (company-lsp--cache-item-candidates (company-lsp--cache-get (company-lsp--prefix-line)))
```

Comment:
Update the parameter name to `prefix-line` in `company-lsp--cache-get`, `company-lsp--candidates-async` and `company-lsp--candidates-sync`

Reply:
Done, (hopefully, I have fixed everything).